packages/uswds-core/src/styles/tokens/units/spacing.scss

```diff
@@ -99,7 +99,7 @@ $system-spacing: (
     // 1400px
   ),
   "special": (
-    0: 0,
+    0: spacing-multiple(0),
```

Comment:
Curious as to why this was changed?

Reply:
@mahoneycm thanks for the reply and apologies for missing it. I must've confused it since the formatting is similar to the testing & review list 🤦.